cmd/ipi-deprovision/ipi-deprovision.sh

```diff
@@ -1,17 +1,38 @@
-#!/bin/sh
+#!/bin/bash
 
-### expirationDate is always with timezone +00:00, eg, "2019-07-23T22:35+0000"
-MY_DATE=$(TZ=":Africa/Abidjan" date '+%Y-%m-%d')
-declare -a regions=("us-east-1" "us-east-2" "us-west-1")
+set -o errexit
+set -o nounset
+set -o pipefail
+
+### cluster_age_cutoff is going to be used as argument in the aws command below
+### to filter out the expired clusters with the tags of the key expirationDate
+### which is always with timezone +00:00, eg, "2019-07-23T22:35+0000"
+cluster_age_cutoff=$(TZ=":Africa/Abidjan" date '+%Y-%m-%d')
+
+if [[ -n "${CLUSTER_TTL-}" ]]; then
```

Comment:
So if this is not set, we will remove all clusters with an expiration date older than now? This is a dangerous default -- we should enforce that this parameter is passed and not use a default

Reply:
Done.